### ISSUE TITLE
evaluation failed for lmoSO

### ISSUE THREAD

**Issue by RuyiLian** [THU-DA-6D-Pose-Group/GDR-Net#18]
Hi, 

When I train GDR-Net on ape of LMO dataset by 
```
./core/gdrn_modeling/train_gdrn.sh configs/gdrn/lmoSO/a6_cPnP_AugAAETrunc_BG0.5_lmo_real_pbr0.1_80e_SO/a6_cPnP_AugAAETrunc_BG0.5_lmo_real_pbr0.1_80e_ape.py 1
```
I get the unexpected output at the end of `log.txt`:
```
core.gdrn_modeling.test_utils WARNING@70: evaluation failed.
core.gdrn_modeling.test_utils INFO@274: =====================================================================
core.gdrn_modeling.test_utils WARNING@316: output/gdrn/lmoSO/a6_cPnP_AugAAETrunc_lmo_real_pbr0.1_80e_SO/ape/inference_model_final/lmo_test/a6-cPnP-AugAAETrunc-BG0.5-lmo-real-pbr0.1-80e-ape-test-iter0_lmo-test-bb8/error:ad_ntop:1 does not exist.
```

Could you suggest how to fix it? Thanks!

**Comment by wangg12** [comment]
Could you share the full log?

**Comment by RuyiLian** [comment]
Please check the attached file, thanks!
[log.txt](https://github.com/THU-DA-6D-Pose-Group/GDR-Net/files/6774181/log.txt)

It is a bit messy because I also tried to run `./core/gdrn_modeling/test_gdrn.sh` several times after training.

**Comment by wangg12** [comment]
Do you have any output for the evaluation subprocess? 
https://github.com/THU-DA-6D-Pose-Group/GDR-Net/blob/main/core/gdrn_modeling/test_utils.py#L69

**Comment by RuyiLian** [comment]
Thanks for your suggestion. I checked the terminal output again and found that I missed one error message: `FileNotFoundError: [Errno 2] No such file or directory: 'datasets/BOP_DATASETS/lmo/test_targets_all.json'`

I downloaded the datasets from the BOP website and I only have `test_targets_bop19.json`. I replaced `test_targets_all.json` by `test_targets_bop19.json` and then the evaluation codes could run successfully. Now I hope to know where I can find the desired `test_targets_all.json`? Thanks!
